mixer/cmd/bundles.go

```diff
@@ -131,5 +164,8 @@ func init() {
 	bundleAddCmd.Flags().BoolVar(&bundleAddFlags.allUpstream, "all-upstream", false, "Add all upstream bundles; takes precedence over bundle list")
 	bundleAddCmd.Flags().BoolVar(&bundleAddFlags.git, "git", false, "Automatically apply new git commit")
 
+	bundleRemoveCmd.Flags().BoolVar(&bundleRemoveFlags.local, "local", false, "Also remove bundle file from local-bundles (irrevocable)")
```

Comment:
Optional: I'd remove this flag for now and see if we miss it.

Reply:
Two reasons I put this in:

1. It's a use case I often have when testing a mix
1. I want some fashion of validity checking. Doing it this way binds removing the local bundle definition file with the removal of that bundle from the Mix Bundle List. If you _don't_ bind these actions (i.e., if you manually `rm` the file on your own), you can be left with a bundle in your mix that doesn't exist. The caveat of this is that if you want to **only** remove the your edited copy (but still have the bundle in your mix, referring to upstream), you'd have to _re-add_ the bundle after running this.

I'm planning to push an update to this that introduces a `--mix` flag that defaults to true. That would mean the behavior described above would be the default, but if you want, you can do `mixer bundle remove editors --local --mix false`. That would keep `editors` in your mix, but just remove your edited copy. If `--mix` is false, I can easily check if that bundle still exists upstream, and error to the user if they're leaving a non-existent bundle in their mix.